processor/src/main/resources/org/mapstruct/ap/internal/model/BeanMappingMethod.ftl

```diff
@@ -13,112 +13,132 @@
         <#assign targetType = returnTypeToConstruct />
     </#if>
     <#list beforeMappingReferencesWithoutMappingTarget as callback>
-    	<@includeModel object=callback targetBeanName=resultName targetType=targetType/>
-    	<#if !callback_has_next>
+        <@includeModel object=callback targetBeanName=resultName targetType=targetType/>
```

Comment:
split the changes here into 2 commits, one to make visible what changed, the other to fix the indenting.